File: Java/ArraySelectionSort.java
import java.util.Scanner;
public class ArraySelectionSort {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		final int SIZE = 5;
		int no[] = new int[SIZE];
		System.out.println("Enter " + SIZE + " numbers below:");
		
		for(int i=0; i<SIZE; i++) {		//Input
			no[i] = scan.nextInt();
		}
		
		for(int i=0; i<SIZE-1; i++) {	//Sorting
			for(int j=i+1; j<SIZE;j++) {
				if(no[i]>no[j]) {
					int t = no[i];
					no[i] = no[j];
					no[j] = t;
				}
			}
		}
		
		System.out.println("The sorted array is:");
		for(int i=0; i<SIZE; i++) {
			System.out.println(no[i]);
		}
	}
}

File: CPP/ArraySelectionSort.cpp
#include<iostream>
using namespace std;
#define SIZE 5
int main()
{
	int no[SIZE];
	cout<<"Enter "<<SIZE<<" numbers below:"<<endl; //cascading put to operator
	for(int i=0; i<SIZE; i++)	//Input
	{
		cin>>no[i];
	}
	
	for(int i=0; i<SIZE-1; i++)	//Sorting
	{
		for(int j=i+1; j<SIZE;j++)
		{
			if(no[i]>no[j])
			{
				int t = no[i];
				no[i] = no[j];
				no[j] = t;
			}
		}
	}
	
	cout<<"The sorted array is:"<<endl;
	
	for(int i=0; i<SIZE; i++)
	{
		cout<<no[i]<<endl;
	}
	return 0;
}
